crt_portal/cts_forms/views.py

```diff
@@ -352,7 +374,8 @@ def get_context_data(self, form, **kwargs):
             })
         elif current_step_name == _('Date'):
             context.update({
-                'page_note': _('It is important for us to know how recently this incident happened. Some civil rights violations must be reported within a certain amount of time.')
+                'page_note': _('It is important for us to know how recently this incident happened. Some civil rights violations must be reported within a certain amount of time.'),
+                'date_question': form.date_question,
```

Comment:
Since the `form` lands in the context, we can skip this and call `form.date_question` directly within `report_date.html`